src/EntryList.js

```diff
@@ -0,0 +1,16 @@
+function EntryList() {
+
+    return (
+        <>
+                <span>
+                    <ul>
+                        Apple
+                        watermelon
```

Comment:
Not very bad, but here you have hardcoded values "Apple" "watermelon" 
I want you to make Entrylist renders any list without having to touch the code inside EntryList 

Currently EntryList shows "Apple" "watermelon" "banana" always 
for example if  I use it in my project 
`
<EntryList />`
`
<EntryList />`

 "Apple" "watermelon" "banana"
 "Apple" "watermelon" "banana"

I want you to remove the hardcoded values ( "Apple" "watermelon" "banana") from EntryList and passe them using `props`

and to do that you can pass these items ( "Apple" "watermelon" "banana") as props to EntryList 

so in the App component you can do 

`,<EntrlyList items={[ "Apple" "watermelon", "banana"]} />`

then you have to do some changes to EntrlyList to make it render list 
I assume you have came across `[].map` and you should be able to use it to show the list

also for each item in the ist you need `li` tag

Reply:
I wanted to use map method 
But first I wanted to see what are my mistakes without using map